### PR TITLE
English: deploy

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
WIP

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Styling etc broken https://defactosoftware.github.io/website-en/ @sn3p

**Comment by sn3p** [comment]
> Styling etc broken https://defactosoftware.github.io/website-en/ @sn3p

Should be fixed when CNAME is properly set

File: Rakefile [hunk 1]
```diff
@@ -79,6 +79,11 @@ namespace :deploy do
...
     deploy :de
   end
+
+  desc "Deploy EN"
+  task :en do
+    deploy :en
+  end
 end
 
 desc "Deploy all locales"
```
Comment (sn3p) [inline Rakefile]:
```
task :deploy => ["deploy:nl", "deploy:de", "deploy:en"]
```

File: README.md [hunk 2]
```diff
@@ -17,17 +17,20 @@ rake serve:en
...
 
 ```bash
 rake deploy:nl
 rake deploy:de
+rake deploy:en
 rake deploy # Deploy all locales
 ```
 
 `:nl` is deployed to [website/tree/gh-pages](https://github.com/DefactoSoftware/website/tree/gh-pages). `:de` is deployed to [website-de/tree/gh-pages](https://github.com/DefactoSoftware/website-de/tree/gh-pages).
+`:en` is deployed to [website-de/tree/gh-pages](https://github.com/DefactoSoftware/website-en/tree/gh-pages).
```
Comment (sn3p) [inline README.md]:
```md
[website-en/tree/gh-pages](...)
```

Also could add an extra space (2 in total) after each sentence here for a line break.

**Comment by RonaldDijkstra** [comment]
#649

File: source/layouts/layout.erb [hunk 3]
```diff
@@ -22,7 +22,8 @@
     <title><%= content_for?(:page_title) ? yield_content(:page_title) : page_title(current_page) %></title>
     <meta name="description" content="<%= meta_description %>">
 
-    <%= tag(:meta, name: "robots", content: current_page.data.robots) if current_page.data.robots -%>
+    <%#= tag(:meta, name: "robots", content: current_page.data.robots) if current_page.data.robots -%>
+    <meta name="robots" content="noindex, nofollow">
```
Comment (sn3p) [inline source/layouts/layout.erb]: Is this a temporary edit?

Comment (RonaldDijkstra) [inline source/layouts/layout.erb]: 👍

**Pull request merged**